pages/telekonseling.tsx

```diff
@@ -0,0 +1,57 @@
+import { GetStaticProps } from "next";
+import { NextSeo } from "next-seo";
+import { BackButton } from "~/components/layout/back-button";
+import {
+  Page,
+  InternalPageContent,
+  InternalPageHeader,
+} from "~/components/layout/page";
+import { Container } from "~/components/ui/container";
+import { TelecounselingSection } from "~/components/telekonseling/telekonseling-link-section";
+import telekonseling, { Telecounseling } from "~/lib/content/telekonseling";
+
+interface TelecounselingPageProps {
+  telekonseling: Telecounseling;
+}
+
+const meta = {
+  title: `Curhat atau Telekonseling`,
+  description: `Informasi mengenai relawan COVID-19 yang memberikan bantuan berupa layanan konsultasi psikologi, kesehatan, agama, dan lainnya yang relevan untuk membantu kesehatan mental terdampak COVID-19.`,
```

Comment:
```suggestion
  title: "Curhat atau Telekonseling",
  description: "Informasi mengenai relawan COVID-19 yang memberikan bantuan berupa layanan konsultasi psikologi, kesehatan, agama, dan lainnya yang relevan untuk membantu kesehatan mental terdampak COVID-19.",
```

Nitpick: use regular string instead of template string if no interpolation exists.